tests/end_to_end/test_helper.cpp

```diff
@@ -51,6 +51,9 @@ static GUID _ebpf_native_provider_id = {/* 5e24d2f5-f799-42c3-a945-87feefd930a7
                                         0x42c3,
                                         {0xa9, 0x45, 0x87, 0xfe, 0xef, 0xd9, 0x30, 0xa7}};
 
+void
+ebpf_platform_wait_for_preemptible_work_items();
+
```

Comment:
Please wrap any statement that can throw with a try catch since an exception in dtor will cause the process to exit.

Reply:
Done.